data/Aditya1404Sal.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Aditya Salunkhe",
+  "type": "personal",
+  "avatar":"https://github.com/Aditya1404Sal.png",
```

Comment:
We no longer need the `avatar` key. So please remove `avatar` key from your `json`. 

```suggestion
```